rustler_mix/lib/rustler.ex

```diff
@@ -16,36 +16,99 @@ defmodule Rustler do
         crate: :my_nif,
         load_data: [1, 2, 3]
 
-  Configuration options:
+  ## Configuration options
 
-    * `:crate` - the name of the Rust crate (as an atom), if different from your
-      `otp_app` value. If you have more than one crate in your project, you will
-      need to be explicit about which crate you intend to use.
+    * `:cargo` - Specify how to envoke the rust compiler. Options are:
+        - `:system` (default) - use `cargo` from the system (must by in `$PATH`)
+        - `{:rustup, <version>}` - use `rustup` to specify which channel to use.
+          Available options include: `:stable`, `:beta`, `:nightly`, or a string
+          which specifies a specific version (i.e. `"1.39.0"`).
+        - `{:bin, "/path/to/binary"}` - provide a specific path to `cargo`.
+
+    * `:crate` - the name of the Rust crate, if different from your `otp_app`
+      value. If you have more than one crate in your project, you will need to
+      be explicit about which crate you intend to use.
+
+    * `:default_features` - a boolean to specify whether the crate's default features
+      should be used.
+
+    * `:env` - Specify a list of environment variables when envoking the compiler.
+
+    * `:features` - a list of features to enable when compiling the crate.
 
     * `:load_data` - Any valid term. This value is passed into the NIF when it is
       loaded (default: `0`)
 
-  Either of the above options can be passed directly into the `use` macro like so:
+    * `:load_from` - This option allows control over where the final artifact should be
+      loaded from at runtime. By default the compiled artifact is loaded from the
+      owning `:otp_app`'s `priv/native` directory. This option comes in handy in
+      combination with the `:skip_compilation?` option in order to load pre-compiled
+      artifacts. To override the default behaviour specify a tuple:
+      `{:my_app, "priv/native/<artifact>"}`. Due to the way `:erlang.load_nif/2`
```

Comment:
Does `:my_app` refer to `:otp_app` or `:crate`? Maybe use the same example as above.

Reply:
`:otp_app`